docs/source/user-guide/models.rst

```diff
@@ -1,2 +1,178 @@
 Models
-======
\ No newline at end of file
+======
+ClimateLearn's model modules are configurable based on the problem
+(forecasting, using the ``ForecastLitModule`` module, and downscaling,
+using the ``DownscaleLitModule`` module) and the desired model archiecture.
+Currently, three deep neural network architectures are supported:
+
+#. Convolutional neural networks: the CNN is a widely used architecture for visual recognition tasks. A constrained version of the standard neural network, CNNs capitalize on knowledge of the input's structure as an image. ClimateLearn suports two popular variants of CNNs:
+
+    #. ResNet: ResNets are a popular variant of CNNs that have been used to achieve weather forecasting for variables such as temperature and geopotential.
+
+    #. U-Net: U-Nets are a CNN varriant that entails both downsampling and upsampling convolutions. Their implementationin ClimateLearn allows users to benchmark U-Net for climate modeling tasks.
+
+#. Vision transformers: the utility of ViTs for representing climate variables is largely under-explored, so ClimateLearn provides an implementation for benchmarking transformers.
+
+Initialization
+--------------
+
+Models are initialized by the ``load_model`` function in the
+``climate_learn.models`` module, which accepts an input for the desired
+architecture (``"vit"``, ``"resnet"``, or ``"unet"``) and the desired
+task (``"forecasting"`` or ``"downscaling"``). The function also accepts
+optional keyword arguments for the model and task optimizer specifically.
+Below is an example of initializing a ViT model for temporal forecasting.
+
+.. code-block:: python
+    :linenos:
+
+    from climate_learn.models import load_model
+    model_kwargs = {
+        "n_blocks": 4
+    }
+    optim_kwargs = {
+        "lr": 1e-4,
+    }
+    model_module = load_model(
+        name="vit",
+        task="forecasting",
+        model_kwargs=None,
+        optim_kwargs=optim_kwargs
+    )
+
+Training
+--------
+
+The ``climate_learn.training`` module provides a ``Trainer`` class for
+fitting and testing models. The trainer is initialized with parameters
+such as the seed, the accelerator, and the maximimum number of epochs.
+
+The module also provides a function for optionally using `Weights and
```

Comment:
Leave out the wandb stuff for now. Unless you've tested and verified that it works, I wouldn't include it in the docs.

Reply:
removed